### PR TITLE
linked URLs

### PR DESCRIPTION
**Pull request by vyaspranjal33**
I've Linked Urls @nikhita

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,14 +3,14 @@ This has useful links to help you get started with contributing to open sources.
 
 ### This repository contains
 
-1. Twitter Accounts which tweets about beginner friendly issues.
-2. List of Website which curates beginner friendly issues.
+1. [Twitter Accounts which tweets about beginner friendly issues](https://github.com/tapasweni-pathak/Getting-Started-With-Contributing-to-Open-Sources/blob/master/Twitter.md)
```
Comment (nikhita) [inline README.md:6]: this can be shortened to just `Twitter.md`. It will act as a relative link and should parse correctly.

File: README.md [hunk 2]
```diff
@@ -3,14 +3,14 @@ This has useful links to help you get started with contributing to open sources.
...
+2. [List of Website which curates beginner friendly issues](https://github.com/tapasweni-pathak/Getting-Started-With-Contributing-to-Open-Sources/blob/master/Websites.md)
```
Comment (nikhita) [inline README.md:7]: same here. just `Websites.md`.

**Review by nikhita — COMMENTED** [review]
Looks good! If you could update with the changes suggested, I'll merge the PR :tada:

**Comment by vyaspranjal33** [comment]
please check @nikhita

**Comment by vyaspranjal33** [comment]
@nikhita  i am closing #7  as required changes have been made and i've opened #10

**Comment by nikhita** [comment]
> @nikhita i am closing #7 as required changes have been made and i've opened #10

https://github.com/OpenSourceHelpCommunity/Getting-Started-With-Contributing-to-Open-Sources#this-repository-contains shows that other links haven't been fixed yet. This PR looks good, please make the changes and this should be good to go.

#10 just updates the first two links, so I going to close it.

@vyaspranjal33 In general, please make any related changes you want to make in the same PR only (as opposed to creating new PRs) - this helps in maintaining the context and comments. :)

**Review by vyaspranjal33 — COMMENTED** [review]
I think PR can be merged now

**Comment by nikhita** [comment]
Awesome, this looks good, @vyaspranjal33! Merging! :tada:

**Pull request merged**